go/pkg/sysdb/coordinator/coordinator_test.go

```diff
@@ -25,6 +25,7 @@ import (
 type APIsTestSuite struct {
 	suite.Suite
 	db                *gorm.DB
+	read_db           *gorm.DB
```

Comment:
do you think it's worth using gorm's [DBResolver](https://gorm.io/docs/dbresolver.html) or is that too much framework overhead?

Reply:
it looked like you could configure it to not use the replica by default and then explicitly use the replica with 

```go
tx := db.Clauses(dbresolver.Use("secondary"), dbresolver.Write).Begin()
```

but not strongly advocating for it